### PR TITLE
fix #83110 whatsnewinpublishing.com

### LINKED ISSUE #83110: whatsnewinpublishing.com
***Description***: https://whatsnewinpublishing.com/this-july-google-chrome-will-make-it-easier-to-bypass-paywalls/ has a newsletter

<details><summary>Screenshots:</summary>

image
image

</details><br/>

<details><summary>Filter:</summary>

image

</details><br/>

### PR DESCRIPTION
**Pull request by wout-codes**
#83110

### CODE REVIEW

File: AnnoyancesFilter/sections/annoyances.txt [hunk 1]
```diff
@@ -2960,7 +2960,8 @@ themoneyhabit.org###blog_subscription-4
 goodreads.com###connectPrompt
 muyinteresante.es###content_newsletter
 bournemouthecho.co.uk###cta1
-investmentwatchblog.com###custom_html-11
+whatsnewinpublishing.com,investmentwatchblog.com###custom_html-11
+whatsnewinpublishing.com###gform_widget-8
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt:2964]: this element is already hidden.

File: SocialFilter/sections/specific.txt [hunk 2]
```diff
@@ -2217,7 +2217,7 @@ family-style.net##.mvp-post-soc-fb
 dnepr.info##.soc-tabs
 vinnytsianews.com##.row > aside.widget-area > aside.widget_fbw_id
 vinnytsianews.com##.row > aside.widget-area > aside.widget-instagram
-vinnytsianews.com##.share-btns-main
+whatsnewinpublishing.com,vinnytsianews.com##.share-btns-main
```
Comment (Alex-302) [inline SocialFilter/sections/specific.txt:2220]: I don't see this element. Found missed button and added `whatsnewinpublishing.com##.themezee-social-sharing`

File: AnnoyancesFilter/sections/annoyances.txt (hunk 1 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/annoyances.txt:2964]: @zloyden Fixed it.

File: SocialFilter/sections/specific.txt (hunk 2 above)
Comment (wout-codes) [reply SocialFilter/sections/specific.txt:2220]: @Alex-302 
image

File: EnglishFilter/sections/specific.txt [hunk 3]
```diff
@@ -270,6 +270,7 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+whatsnewinpublishing.com##.wp-block-spacer
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:273]: It doesn't contain ad - remove the rule..

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/specific.txt (hunk 3 above)
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:273]: Ok.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**